mlflow/exceptions.py

```diff
@@ -37,7 +37,7 @@ def serialize_as_json(self):
 class RestException(MlflowException):
     """Exception thrown on non 200-level responses from the REST API"""
     def __init__(self, json):
-        error_code = json['error_code']
+        error_code = json.get('error_code')
```

Comment:
Did you mean for this to be json.get('error_code', INTERNAL_ERROR) ?

Reply:
Currently I left the default value to MlflowException's default value( None -> INTERNAL_ERROR). I can make it explicit or, if there is a better error_code, we can set it here as you described.

Reply:
I think both options are fine.
One difference is, if there is no message nor error_code in the json then the exception will have no message.